test/ng/directive/ngModelOptionsSpec.js

```diff
@@ -480,15 +480,37 @@ describe('ngModelOptions', function() {
         expect($rootScope.checkbox).toBeUndefined();
```

Comment:
Not directly related to this PR, but `checkbox` should be `name` (here and above).